README.md

```diff
@@ -10,7 +10,7 @@ different criteria such as scope, author, year, etc.
 Requirements
 ------------
 
-* Django 2.2
+* Django 3.2
```

Comment:
We could do Django 2.2-3.2, since it still works in Django 2.2, but perhaps since it is out of support we should leave what you have? Not sure what best practice is.

Reply:
It does still work with Django 2.2. I can add back the support for it and do as you say, Django 2.2-3.2.